crates/db_views/notification/src/impls.rs

```diff
@@ -130,6 +153,9 @@ impl NotificationView {
       .left_join(my_post_actions_join)
       .left_join(my_person_actions_join)
       .left_join(my_comment_actions_join)
+      .left_join(admin_add::table)
+      .left_join(mod_add_to_community::table)
+      .left_join(admin_ban::table)
```

Comment:
Moving these above the `my_` joins will also probably compile faster.

Reply:
Good thinking, now only 26s. Didnt expect the order to make so much difference.

Reply:
Actually the slowness only happens when compiling tests, the production code compiles in 25s. Very strange as its not doing anything unusual. This makes it very impractical to write more test cases.